packages/knip/src/plugins/vitest/index.ts

```diff
@@ -48,7 +48,7 @@ const findConfigDependencies = (localConfig: ViteConfig, options: PluginOptions)
   return [...[...environments, ...reporters, ...coverage].map(toDependency), ...setupFiles, ...globalSetup];
 };
 
-const getConfigs = async (localConfig: ViteConfigOrFn | VitestWorkspaceConfig) => {
+export const getConfigs = async (localConfig: ViteConfigOrFn | VitestWorkspaceConfig) => {
```

Comment:
Perhaps we can import the babel function here and handle it from the vitest plugin?